prime-router/src/main/kotlin/azure/SubmissionsFacade.kt

```diff
@@ -66,7 +66,14 @@ class SubmissionsFacade(
             SubmissionAccess.SortColumn.CREATED_AT
         }
 
-        return findSubmissions(organizationName, order, column, offset, toEnd, pageSize, showFailed)
+        /* This will swap the order of the date-time offsets IF both are present
```

Comment:
Why swap `toEnd` and `offset`?

Reply:
This was what I was telling you about earlier regarding the date swapping. This'll measure if the user's start range is less than the end range, and swap them. If they don't, they get nothing back because the `between` clause of the query has an invalid date-range. This basically just means that the "Start" and "End" aren't strictly defined, and a user could make their start date the closer-to or further-from current time of the two values, and ensures the user won't ever get an empty result due to a bad date-range

Reply:
Yep it works when pointing to staging now